packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -26,7 +26,7 @@ export class BoosterGraphQLDispatcher {
 
   public constructor(private config: BoosterConfig, private logger: Logger) {
     this.readModelDispatcher = new BoosterReadModelsReader(config, logger)
-    this.graphQLSchema = GraphQLGenerator.build(config, logger).generateSchema()
+    this.graphQLSchema = GraphQLGenerator.generateSchema(config, logger)
```

Comment:
I think that in future changes we should move the logger to the Booster singleton so it becomes accessible as `Booster.logger`. Once that's done we could avoid the need of injecting these instances all over the place, as the `Booster` instance is globally accessible.